src/Sentry/Internal/BackgroundWorker.cs

```diff
@@ -46,7 +46,7 @@ internal class BackgroundWorker : IBackgroundWorker, IDisposable
             _shutdownSource = shutdownSource ?? new CancellationTokenSource();
             _queuedEnvelopeSemaphore = new SemaphoreSlim(0, _maxItems);
 
-            WorkerTask = Task.Run(async () => await WorkerAsync().ConfigureAwait(false));
+            WorkerTask = Task.Run(() => WorkerAsync());
```

Comment:
```suggestion
            WorkerTask = Task.Run(WorkerAsync);
```
Might as well simplify this further